pkg/ebpf/tracee.go

```diff
@@ -1165,3 +1155,27 @@ func (t *Tracee) updateKallsyms() error {
 	t.kernelSymbols = kernelSymbols
 	return nil
 }
+
+// attachSelfUprobe attaches a Uprobe to itself to invoke an eBPF program from userspace
+func (t *Tracee) attachSelfUprobe(progFn string, symbolName string) error {
```

Comment:
This logic can be used in probes.go as part of integration.

Reply:
https://github.com/rafaeldtinoco/tracee/commit/19f112bdc5d6f7694a61025980edeebcfcbd2fba

Since I recently touched that code, I created the needed change if you want to pick that commit and merge into your PR and push it.